virtwho.py

```diff
@@ -25,20 +25,62 @@
 import time
 from multiprocessing import Event, Queue
 import json
+
 import atexit
 from Queue import Empty
 from httplib import BadStatusLine
 
 from daemon import daemon
-from virt import Virt, DomainListReport, HostGuestAssociationReport, ErrorReport
+from virt import Virt, DomainListReport, HostGuestAssociationReport, ErrorReport, Hypervisor
 from manager import Manager, ManagerError, ManagerFatalError
+from manager.managerprocess import ManagerProcess
+from manager.subscriptionmanager import SubscriptionManager
 from config import Config, ConfigManager, InvalidPasswordFormat
 from password import InvalidKeyFile
+from datetime import datetime, timedelta
+from heapq import *
 
 import log
 
 from optparse import OptionParser, OptionGroup, SUPPRESS_HELP
 
+# Default interval for jobs to be delayed
+DefaultJobInterval = 60
```

Comment:
I think 60 seconds is quite a lot, esp. for first run with small amount of data. In previous version the submission was almost immediate, now it just 60 seconds even when no association is sent.